core/src/main/java/hudson/node_monitors/AbstractNodeMonitorDescriptor.java

```diff
@@ -53,13 +54,15 @@
  * @author Kohsuke Kawaguchi
  */
 public abstract class AbstractNodeMonitorDescriptor<T> extends Descriptor<NodeMonitor> {
+    private static int PERIOD_MINUTES = SystemProperties.getInteger(AbstractNodeMonitorDescriptor.class.getName() + ".periodMinutes", 60);
```

Comment:
```suggestion
    private static long PERIOD = SystemProperties.getLong(AbstractNodeMonitorDescriptor.class.getName() + ".period", TimeUnit.MINUTES.toMillis(5));
```
The type expected for scheduling is long and you avoid converting in both constructors: `AbstractNodeMonitorDescriptor()` and `AbstractNodeMonitorDescriptor(Class<? extends NodeMonitor> clazz)`